components/places/src/places.udl

```diff
@@ -21,10 +21,21 @@ enum ConnectionType {
     "Sync", // ideally we wouldn't need to expose this and should guard against it being used (it's internal only)
 };
 
+interface SqlInterruptHandle {
+     void interrupt();
+};
+
 interface PlacesApi {
     [Throws=PlacesError]
     PlacesConnection new_connection(ConnectionType conn_type);
 
+    //NOTE: this doesn't appear to be actively used and can probably remove
```

Comment:
In theory it *should* be used by ios so they can cancel a long running sync on shutdown, but yeah, for now, this is fine (maybe update the comment to also say the sync manager will typically arrange to interrupt syncs or something)